tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -23,252 +23,185 @@ TEST_BUCKET_NAME=$1
-sudo umount $MOUNT_DIR
-
-# Run test with persistent mounting
+# Run test with persistent mounting. (flags: --implicit-dirs=true)
 mount.gcsfuse $TEST_BUCKET_NAME $MOUNT_DIR -o implicit_dirs=true
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
 sudo umount $MOUNT_DIR
 
+# Run test with static mounting. (flags: --implicit-dirs=false)
 gcsfuse --implicit-dirs=false $TEST_BUCKET_NAME $MOUNT_DIR
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
 sudo umount $MOUNT_DIR
 
-# Run test with persistent mounting
+# Run test with persistent mounting. (flags: --implicit-dirs=false)
 mount.gcsfuse $TEST_BUCKET_NAME $MOUNT_DIR -o implicit_dirs=false
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
 sudo umount $MOUNT_DIR
 
-# Run integration tests for operations with --only-dir mounting.
-gcsfuse --only-dir testDir --enable-storage-client-library=true --implicit-dirs=true $TEST_BUCKET_NAME $MOUNT_DIR
-GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
-sudo umount $MOUNT_DIR
-
-# Run test with persistent mounting
-mount.gcsfuse $TEST_BUCKET_NAME $MOUNT_DIR -o only_dir=testDir,enable_storage_client_library=true,implicit_dirs=true
+# Run tests with --only-dir mounting. (flags: --implicit-dirs=true)
+gcsfuse --only-dir testDir --implicit-dirs=true $TEST_BUCKET_NAME $MOUNT_DIR
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
 sudo umount $MOUNT_DIR
 
-gcsfuse --only-dir testDir --enable-storage-client-library=false  $TEST_BUCKET_NAME $MOUNT_DIR
-GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
-sudo umount $MOUNT_DIR
-
-# Run test with persistent mounting
-mount.gcsfuse $TEST_BUCKET_NAME $MOUNT_DIR -o only_dir=testDir,enable_storage_client_library=false
-GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
-sudo umount $MOUNT_DIR
-
-gcsfuse  --only-dir testDir --implicit-dirs=true $TEST_BUCKET_NAME $MOUNT_DIR
-GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
-sudo umount $MOUNT_DIR
-
-# Run test with persistent mounting
+# Run tests with persistent mounting. (flags: --implicit-dirs=true, --only-dir=testDir)
 mount.gcsfuse $TEST_BUCKET_NAME $MOUNT_DIR -o only_dir=testDir,implicit_dirs=true
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
 sudo umount $MOUNT_DIR
 
+# Run tests with --only-dir mounting. (flags: --implicit-dirs=false)
```

Comment:
should this be called static mounting and move --only-dir to inside brackets as other headings.